src/qml/MessageLog.qml

```diff
@@ -144,15 +144,76 @@ Page {
         id: submitLog
         Layout.fillWidth: true
         text: qsTr( 'Send application log' )
-        visible: qfieldSettings.enableInfoCollection && platformUtilities.capabilities & PlatformUtilities.SentryFramework
+        visible: true//qfieldSettings.enableInfoCollection && platformUtilities.capabilities & PlatformUtilities.SentryFramework
 
         onClicked: {
-            iface.sendLog("Manual log submission")
-            displayToast(qsTr("Your application log is being sent"))
+            applicationLogDialog.open()
         }
     }
   }
 
+  Dialog {
+    id: applicationLogDialog
+    title: qsTr("Send application log")
+    focus: true
+    font: Theme.defaultFont
+
+    x: ( mainWindow.width - width ) / 2
+    y: ( mainWindow.height - height - 80 ) / 2
+
+    onAboutToShow: {
+      appliationLogInput.text = ''
+    }
+
+    Column {
+      width: childrenRect.width
+      height: childrenRect.height
+      spacing: 10
+
+      TextMetrics {
+        id: applicationLogLabelMetrics
+        font: applicationLogLabel.font
+        text: applicationLogLabel.text
+      }
+
+      Label {
+        id: applicationLogLabel
+        width: mainWindow.width - 60 < applicationLogLabelMetrics.width ? mainWindow.width - 60 : applicationLogLabelMetrics.width
+        text: qsTr("Type a message to attach to your application log:")
```

Comment:
Message or details should be enough.

And somewhere a label explaining "this will send a log of your current session to the development team. You only need to do this when you are asked for it." Or similar?